### DISCUSSION TITLE
FR: Plugin Store Page layout refinement with common locations for UI, example given:

### DISCUSSION THREAD

**Discussion by bennash** [craftcms/cms#8770] [Ideas] [answered]
### NEW UI Locations
The Plugin page in the Store could use some layout adjustments. The Install buttons are often pushed down the page with long content above. This UI could be anchored up top. A common location for this UI would be helpful with muscle memory, always navigating to this ui's location. 

Many of the informational pieces could be moved to the sidebar below the Documentation and Repo links. 

### My suggested layout ideas:
https://codepen.io/bennash/pen/LYEqpLX?editors=1000

**Comment by brandonkelly** [comment, marked as answer]
We significantly improved the plugin detail page layout in Craft 4.
